### DISCUSSION TITLE
label  selected

### DISCUSSION THREAD

**Discussion by yuenxillar** [emilk/egui#5103] [Q&A]
![image](https://github.com/user-attachments/assets/cb1e679a-3f18-4240-9e7a-4c5ae223a8a9)
May I ask how I can globally disable it? When hovering over text, the mouse style becomes text selection

**Comment by rustbasic** [comment]
It may be slightly different depending on the egui version, but it is roughly as follows:

```
            ui.ctx().style_mut(|style| {
                style.interaction.selectable_labels = false;
            });
```
